### PR TITLE
Add missing metadata for schema.org

### LINKED ISSUE #5424: Add schema.org metadata to STYs
**Is your feature request related to a problem? Please describe.**
We would like to decorate our STY pages with Schema.org metadata as we do in our Article pages.

**Describe the solution you'd like**
Add/amend the schema.org fields for STYs to match what is in the tables.

Schema.org metadata is processed through the LinkedData container which can be found [here](https://github.com/bbc/simorgh/blob/latest/src/app/containers/LinkedData/index.jsx). Will need to check if there are any pieces of metadata that aren't needed for STY pages.

Structure of the schema data can be found on the [schema.org](https://schema.org/docs/schemas.html) website.

**Note**: all remaining fields should be the same as for the article page, _including the presence of `videoObject `or `audioObject `markup where AV is present in an article_. Further issues/refinement may be needed if `videoObject `and `audioObject `are not present.

[See structured data tool](https://search.google.com/structured-data/testing-tool#url=https%3A%2F%2Fwww.bbc.co.uk%2Fnews%2Farticles%2Fce9v85jlrzzo). With the code snippet section you just have to use the JSON that's generated on the page.

Based on example: https://www.bbc.com/mundo/noticias-internacional-51266689

| Field        |  Value           |  Example  | Notes  |
| ------------- | ----------- | ---------- | ---------- |
| description |	Content block Role: Introduction | El controvertido Brexit llega finalmente con la salida oficial de Reino Unido de la Unión Europea (UE) este 31 de enero, aunque la incertidumbre respecto a sus efectos prácticos todavía está lejos de ser despejada. |  | 
| headline | Promo Headlines Headline | Brexit: qué cambiará para visitar, trabajar y estudiar en Reino Unido tras la salida del país de la Unión Europea | | 
| type | categoryName | Use this mapping to transform category to the schema-appropriate value: <br>'News' = 'ReportageNewsArticle', <br>'Explainer' = 'BackgroundNewsArticle', <br>'Analysis' = 'AnalysisNewsArticle', <br>'Opinion' = 'OpinionNewsArticle', <br>'Feature' = 'ReportageNewsArticle',<br>'Infographic' = 'ReportageNewsArticle', <br>'Interactive' = 'ReportageNewsArticle', <br>'Newsbeat Investigates' = 'ReportageNewsArticle', <br>'Quiz' = 'ReportageNewsArticle', <br>'Summary' = 'ReportageNewsArticle', <br>'You what?' = 'ReportageNewsArticle', <br>'Advertiser Content' = 'AdvertiserContentArticle', <br>'Review' = 'ReviewNewsArticle', <br>'Ask the Audience' = 'AskPublicNewsArticle' | | 

The following are sub-fields of **image**, which is an [ImageObject](https://schema.org/ImageObject).

| Field        |  Value           |  Example  | Notes |
| ------------- | ----------- | ---------- |---------- | 
| @type |	ImageObject |    |  | 
| url | Promo Index Image | https://ichef.bbci.co.uk/news/208/cpsprodpb/17D42/production/_110620679_gettyimages-542984918.jpg  | Fallback to service logo if no image available   | 
| width	| Width of the image|	 | |
| height	| Height of the image |	 | |

**Describe alternatives you've considered**

**Testing notes**

Just update/check existing unit tests/snapshots we have for the LinkedData component. Ensure that other page types won't break with this implementation.

Use previously mentioned validation tools to test the implementation.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**
See https://github.com/bbc/simorgh/issues/5159 for similar work on PGLs.

### PR DESCRIPTION
**Pull request by FK78**
Resolves #5424 

**Overall change:** 
_Add missing metadata for schema.org._

**Code changes:**
- _Updated snapshots for `CpsSty` container._
- _Created categoryMap._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**
- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/pages/CpsSty/categoryMap/index.test.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import categoryMap from './index';
+
+test('should assign the correct value', () => {
```
Comment (andrewscfc) [inline src/app/pages/CpsSty/categoryMap/index.test.js]:
```suggestion
test('should return correct mapping when category recognised', () => {
```

File: src/app/pages/CpsSty/categoryMap/index.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+  expect(categoryMap('Analysis')).toEqual('AnalysisNewsArticle');
+});
+
+test('should assign NewsArticle when category is null', () => {
```
Comment (andrewscfc) [inline src/app/pages/CpsSty/categoryMap/index.test.js]:
```suggestion
test('should return NewsArticle when category is null', () => {
```

File: src/app/pages/CpsSty/categoryMap/index.test.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  expect(categoryMap('')).toEqual('NewsArticle');
+});
```
Comment (andrewscfc) [inline src/app/pages/CpsSty/categoryMap/index.test.js:13]: Perhaps add a test for when the category is not recognised: `expect(categoryMap('random category')).toEqual('NewsArticle');`

File: src/app/pages/CpsSty/__snapshots__/index.test.jsx.snap [hunk 4]
```diff
@@ -2349,7 +2349,7 @@ exports[`CPS STY Page snapshots should match snapshot for STY 1`] = `
...
...ext":"http://schema.org","@type":"NewsArticle","url":"https://www.test.bbc.com/pidgin/world-23252817","publisher":{"@type":"NewsMediaOrganization","name":"BBC News Pidgin","publishingPrinciples":"https://www.bbc.com/news/help-41670342","logo":{"@type":"ImageObject","width":1024,"height":576,"url":"https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png"}},"image":{"@type":"ImageObject","width":1024,"height":576,"url":"https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png"},"thumbnailUrl":"https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png","mainEntityOfPage":{"@type":"WebPage","@id":"https://www.test.bbc.com/pidgin/world-23252817","name":"STY - Blood pressure drugs dey work better"},"headline":"STY - Blood pressure drugs dey work better","description":"This is a test asset used in Simorgh, please do not edit it without asking.","datePublished":"2019-10-23T12:39:30.000Z","dateModified":"2019-10-23T12:39:32.000Z","author":{"@type":"NewsMediaOrganization","name":"BBC News Pidgin","logo":{"@type":"ImageObject","width":1024,"height":576,"url":"https://news.files.bbci.co.uk/ws/img/logos/og/pidgin.png"},"noBylinesPolicy":"https://www.bbc.com/news/help-41670342#authorexpertise"}}
```
Comment (andrewscfc) [inline src/app/pages/CpsSty/__snapshots__/index.test.jsx.snap:2352]: Does the pigin test fixture not include one of the recognised categories: `"@type":"NewsArticle"`?

File: src/app/pages/CpsSty/categoryMap/index.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    Interactive: 'ReportageNewsArticle',
+    'Newsbeat Investigates': 'ReportageNewsArticle',
+    Summary: 'ReportageNewsArticle',
+    'You what?': 'ReportageNewsArticle',
+    Explainer: 'BackgroundNewsArticle',
+    Analysis: 'AnalysisNewsArticle',
+    Opinion: 'OpinionNewsArticle',
+    'Advertiser Content': 'AdvertiserContentArticle',
+    Review: 'ReviewNewsArticle',
+    'Ask the Audience': 'AskPublicNewsArticle',
```
Comment (andrewscfc) [inline src/app/pages/CpsSty/categoryMap/index.js:16]: Found one missing category there:
>'Quiz' = 'ReportageNewsArticle', 

You may want to check again yourself: https://github.com/bbc/simorgh/issues/5424#issue-560575200

File: src/app/pages/CpsSty/__snapshots__/index.test.jsx.snap (hunk 4 above)
Comment (FK78) [reply src/app/pages/CpsSty/__snapshots__/index.test.jsx.snap:2352]: Nope, the `pidginPageData` has an empty object of `passport`.

Comment (andrewscfc) [reply src/app/pages/CpsSty/__snapshots__/index.test.jsx.snap:2352]: You may want to check this fixture data is up to date, the issue doesn't seem to suggest it's that common that the data will be missing, it doesn't even mention the possibility. Maybe @jimjohnsonrollings or @lizcameron could shed some light?

**Review by andrewscfc — COMMENTED** [review]
LGTM aside from maybe the fixture data (that could be correct though). I'd rather not approve as I've paired with you on this to some extent. Great work though 😄

File: src/app/pages/CpsSty/categoryMap/index.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    'You what?': 'ReportageNewsArticle',
+    Quiz: 'ReportageNewsArticle',
+    Explainer: 'BackgroundNewsArticle',
+    Analysis: 'AnalysisNewsArticle',
+    Opinion: 'OpinionNewsArticle',
+    'Advertiser Content': 'AdvertiserContentArticle',
+    Review: 'ReviewNewsArticle',
+    'Ask the Audience': 'AskPublicNewsArticle',
+  };
+  return dictonary[category] ? dictonary[category] : 'NewsArticle';
```
Comment (chris-hinds) [inline src/app/pages/CpsSty/categoryMap/index.js:18]: with ES2020 support in babel you should be able to shorten this.
```suggestion
  return dictonary[category] ?? 'NewsArticle';
```

**Review by chris-hinds — APPROVED** [review]
👍 nice work

File: src/app/pages/CpsSty/categoryMap/index.js (hunk 6 above)
Comment (chris-hinds) [reply src/app/pages/CpsSty/categoryMap/index.js:18]: Actually, this is experimental syntax apparently, so we might have ti remove it :(

```
 This experimental syntax requires enabling the parser plugin: 'nullishCoalescingOperator' (18:29)
```

**Review by ryanmccombe — APPROVED** [review]
Nice 👍

File: src/app/pages/CpsSty/__snapshots__/index.test.jsx.snap (hunk 4 above)
Comment (jimjohnsonrollings) [reply src/app/pages/CpsSty/__snapshots__/index.test.jsx.snap:2352]: It was published by Drew in October - I'm surprised he was able to publish it without a category. Will add one now.

Comment (jimjohnsonrollings) [reply src/app/pages/CpsSty/__snapshots__/index.test.jsx.snap:2352]: It's updated in Ares, if you want to update the fixture from it :) - /pidgin/world-23252817

**Pull request merged**